src/main/java/org/zalando/nakadi/config/ValidatorConfig.java

```diff
@@ -39,7 +38,6 @@ public SchemaEvolutionService schemaEvolutionService() throws IOException {
 
         final List<SchemaEvolutionConstraint> schemaEvolutionConstraints = Lists.newArrayList(
                 new CompatibilityModeChangeConstraint(),
-                new CompatibleSchemaChangeConstraint(),
```

Comment:
I also removed the test because of it. the upstream branch is not up to date with the latest changes.
@rcillo can elaborate on that :)